R/deps.R

```diff
@@ -109,7 +109,7 @@ dev_package_deps <- function(pkg = ".", dependencies = NA,
   deps <- unlist(lapply(parsed, `[[`, "name"), use.names = FALSE)
 
   if (is_bioconductor(pkg)) {
-    check_suggested("BiocInstaller")
+    check_bioconductor()
```

Comment:
It's slightly annoying that there's an interactive prompt here, because it might get triggered arbitrarily later after starting `revdep_check()`. But I don't know how to avoid it